lib/src/login/login.dart

```diff
@@ -191,13 +196,44 @@ class _LoginState extends State<Login> {
     List<TextSpan> textParts = [];
     textParts.add(TextSpan(text: formattedAgreeString.substring(spanBoundary, termsConditionsStartIndex)));
     spanBoundary = termsConditionsStartIndex;
-    textParts.add(UrlTextSpan(url: null, text: termsAndConditions));
+    textParts.add(TextSpan(
+      text: termsAndConditions,
+      style: TextStyle(color: accent, fontWeight: FontWeight.bold),
+      recognizer: TapGestureRecognizer()
+        ..onTap = () {
+          _navigation.push(
+              context,
+              MaterialPageRoute(
+                builder: (context) => Privacy(
+                  htmlText: _terms,
+                ),
+              ));
+        },
+    ));
```

Comment:
Please reuse **UrlTextSpan** and extend that class to be able to handle also those asset loading requests. So we can use it for termsAndConditions and privacyPolicy loading.
An idea could be adding another paramter `String asset` and making `String url` not required. Within the class we then check which one is set and provide the appropriate view (we should throw an exception if non is set). Later we could extend that class even further, but for now this should be fine.